JSDemos/Demos/PivotGrid/DrillDown/description.md

```diff
@@ -1 +1,5 @@
-Drill-down support within the PivotGrid component allows you to&nbsp;retrieve the list of&nbsp;records used to&nbsp;calculate a&nbsp;specific summary value. To&nbsp;see drill downs in&nbsp;action, click a&nbsp;summary cell and view the records that formed the basis of&nbsp;the summary value.
\ No newline at end of file
+PivotGrid supports the drill down operation that allows users to retrieve the list of facts used to calculate a specific summary value. 
```

Comment:
This and two following sentences all have the same ending (retrieve the list of facts used to calculate a specific summary value; access underlying facts for a summary value; a list of facts that form the basis of the summary value.). Can we simplify or shorten this text ?